docs/jina-ai-cloud/index.md

```diff
@@ -20,7 +20,7 @@ Executor Hub is an Executor marketplace that allows you to share, explore and te
 :link: ../concepts/jcloud/index
 :link-type: doc
 
-JCloud is a free CPU/GPU hosting platform for Jina projects.
+JCloud is a CPU/GPU hosting platform for Jina projects.
```

Comment:
```suggestion
JCloud is a cost-efficient hosting platform specifically designed for Jina projects.
```

Reply:
sorry should have done a suggestion right from the start. After this change it can be merged from my side